### PR TITLE
There should be more space visually between conditions of sale checkbox and place bid button

### PR DESCRIPTION
**Pull request by yuki24**
finishes https://artsyproduct.atlassian.net/browse/PURCHASE-181

 * Prefer `Credit card` over `Credit Card`
 * The margin between the checkbox and the Place bid button should be 20px when alreadt agreed and 40px when needs to show a checkbox
 * Changed the margin for each row to be 20px so all the elements will be accommodated well in iPhone SE's smaller screen
# Screenshots:

screen_shot_2018-06-27_at_2_53_46_pm

screen_shot_2018-06-27_at_3_43_02_pm

#skip_new_tests

### CODE REVIEW

**Comment by sweir27** [comment]
👏 💅 

I think this will also close this ticket: https://artsyproduct.atlassian.net/browse/PURCHASE-213 😄

**Comment by yuki24** [comment]
Unfortunately this won't completely close https://artsyproduct.atlassian.net/browse/PURCHASE-213 as the main issue is in Eigen adding an extra space to the top of the modal view controller.

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 * Removes unnecessary white spaces in the bid result screen - yuki24
 * Updates the text style and order for the auction timer - yuki24
 * Registration screen sends mutation for creating a bidder/credit card - sweir27
+* There should be more space visually between conditions of sale checkbox and place bid button - yuki24
```
Comment (erikdstock) [inline CHANGELOG.md]: I think this should go under `### master`

Comment (yuki24) [reply CHANGELOG.md]: Right, a new version just case out and this new entry was left in the wrong spot.

**Pull request merged**